codespell_lib/data/dictionary.txt

```diff
@@ -36506,6 +36613,7 @@ wasnt'->wasn't
 wasnt->wasn't
 wasnt;->wasn't
 wass->was
+wast->waste
```

Comment:
`wast` seem like a valid word. I suggest you drop this. The checks failed also because of this.
Maybe KIV for future dictionary that is not enabled by default.
* https://www.ldoceonline.com/dictionary/wast
* https://dictionary.cambridge.org/dictionary/english/wast
* https://www.merriam-webster.com/dictionary/wast